app/assets/stylesheets/frontend/views/_history-people.scss

```diff
@@ -168,3 +143,24 @@
 
   }
 }
+
+.aspect-ratio-3\:2 {
+  display: block;
+  height: 0;
+  overflow: hidden;
+  padding: 66% 0 0 0;
+  position: relative;
+  width: 100%;
```

Comment:
Nitpick: width 100% is default behaviour for display block, so this line isn't needed.

Reply:
➖ Removed

Reply:
➖  Removed